src/commands/Moderation/multiunban.ts

```diff
@@ -50,10 +50,10 @@ export default class extends Command {
 		for (let i = 0; i < targets.length; i++) {
 			if (targets[i] === undefined) continue;
 			await this.client.bulbutils.sleep(massCommandSleep);
+			const targetID: Snowflake = targets[i].replace(NonDigits, "")
 
-			let target;
 			let infID: number;
-			target = await this.client.bulbfetch.getUser(targets[i].replace(NonDigits, ""));
+			let target = await this.client.bulbfetch.getUser(targetID);
```

Comment:
Couldn't it be a `const`?

Reply:
Yes definitely, thank you for pointing it out